src/main/java/fi/metatavu/metamind/weka/RecommendationData.java

```diff
@@ -0,0 +1,108 @@
+package fi.metatavu.metamind.weka;
+
+import java.util.ArrayList;
+
+import weka.core.Attribute;
+import weka.core.DenseInstance;
+import weka.core.Instance;
+import weka.core.Instances;
+
+/**
+ * Helper class for converting data
+ * 
+ * @author Simeon Platonov
+ */
+public class RecommendationData {
+	ArrayList<WekaRecommendationItem> trainingItems;
+	ArrayList<WekaRecommendationItem> itemsToRecommend;
+	ArrayList<Attribute> attributeInfo;
+	WekaRecommendationItem[] items;
+	Instances trainingSet;
+	Instances recommendationSet;
+	
+	/**
+	 * 
+	 * @param items - All rated and unrated items
+	 */
+	public RecommendationData(WekaRecommendationItem[] items) {
+		this.items = items;
+	}
+	
+	/**
+	 * Converts text attributes to an integer array of 0s and 1s
+	 */
+	void convertAttributes() {
```

Comment:
Missing method access modifiers (in all methods of this class)

Reply:
Access modifiers added